packages/agent/src/did-api.ts

```diff
@@ -231,7 +231,7 @@ export class AgentDidApi<TKeyManager extends AgentKeyManager = AgentKeyManager>
     // Resolve the DID document.
     const { didDocument, didResolutionMetadata } = await this.resolve(didUri);
     if (!didDocument) {
-      throw new Error(`DID resolution failed for '${didUri}': ${didResolutionMetadata.error}`);
+      throw new Error(`DID resolution failed for '${didUri}': ${JSON.stringify(didResolutionMetadata)}`);
```

Comment:
This has nothing to do with this PR, however this is intermittently failing in CI so I just added it here instead of cutting a new release specifically for it.